move main pages from v4.loopback.io

dhmlau opened the pull request:
Move the main pages from v4.loopback.io, so that the loopback.io will be showing the LB4 content.

### User Experience
When user going to loopback.io, it shows the same content as v4.loopback.io.  All the pages accessible from the navbar (at the top) will be the new LB4 content.
![Screen Shot 2019-05-29 at 2 50 31 PM](https://user-images.githubusercontent.com/25489897/58583049-1f2bb300-8221-11e9-9262-1e83cfd5f3b4.png)

- `Docs` at the nav bar will be pointing to `/doc/lb4`.  The only way to get to LB3 documentation is either:
  1. you go to the docs page and click `LoopBack`, that will bring you to the list of LB versions
  2. at the footer > Documentation link will bring you to  https://loopback.io/doc/ which shows all the versions. 

After this PR has merged, we can redirect v4.loopback.io to loopback.io.

### Discussion
@strongloop/loopback-maintainers, there are a few topics that I'd like to get your inputs:
1. Github stars shown on loopback.io
I've changed it to show the github star for `loopback-next` repo.  As you might aware already, it's a much lower number.  I think it makes sense to use the LB4 repo github star even though it's less impressive at this moment.

2. LoopBack 3 pages
For the following LB3 pages, I've "relocated" it to /lb3/* url. 
i.e. loopback.io/getting-started -> loopback.io/lb3/getting-started
If we still want users to access those pages, I'd like to propose to have the banner at the top adding to say if you want to access the LoopBack 3 pages, go to `/lb3/` link.  But i'm not sure if it's necessary.

dhmlau committed:
move files from v4.loopback.io

dhmlau commented on index-lb3.html:
```diff
@@ -0,0 +1,202 @@
+---
```
rename it from `index.html`.

dhmlau commented on index.html:
```diff
@@ -1,201 +1,265 @@
+
+    <!--Header alert -->
+    <div class="header-alert">
+        <p class="centered-regular">LoopBack 4 GA (General Availability) has been released in October 2018, read more in <a href="http://strongloop.com/strongblog/loopback-4-ga" target="_blank">the announcement post.</a> Click <a href="/lb3">here</a> for LoopBack 3 pages.</p>
```
I've edited the alert to show how to access the LB3 pages.  Not sure if it's necessary.

b-admike reviewed (approved):
I am not too familiar with the changes, but from the demo you gave the team and the direction this PR is taking, LGTM. It makes sense to have the landing page of `loopback.io` point to the current release of our framework 💪.

nabdelgadir reviewed (approved):
Two things:
1) favicon is still the green one for the v4 pages
2) clicking on the LoopBack symbol on the green `/lb3` pages leads back to the main lb4 page

Other than that LGTM 👍

dhmlau commented:
> favicon is still the green one for the v4 pages

It seems to be working for me, i.e. showing the LB4 logo as the favicon.  Do you mind opening the browser in incognito mode to try it out again? Thanks.

> clicking on the LoopBack symbol on the green /lb3 pages leads back to the main lb4 page

I think it makes sense to bring back to the main page loopback.io when clicking on the old LB logo?

nabdelgadir commented:
@dhmlau 

> It seems to be working for me, i.e. showing the LB4 logo as the favicon. Do you mind opening the browser in incognito mode to try it out again? Thanks.

Tried it on Firefox and it's the right one 👍 

> I think it makes sense to bring back to the main page loopback.io when clicking on the old LB logo?

How would you go back to the main lb3 page? I'm fine with how it is now, but just wondering though.

dhmlau commented:
I think it will go back to the same way how it brings you to the /lb3.  An alternative is to put "LoopBack 3" in the navbar, which may not be a bad idea. 
There was a comment about combining `Resources` and `Docs`, if that's the case, it can give us more room. let me try and post a screen cap.

dhmlau commented:
@nabdelgadir, I think i like this one better.
- Have `LoopBack 3` in the navbar
- Remove `Docs` from navbar
- Alert: keep it only contains the GA blog. 

![Screen Shot 2019-05-30 at 1 39 03 PM](https://user-images.githubusercontent.com/25489897/58651856-4bf2cf80-82e0-11e9-9a74-e0fc524c99df.png)

WDYT?

nabdelgadir commented:
@dhmlau LGTM 👍

dhmlau commented:
@slnode test please

raymondfeng commented:
@dhmlau Some quick feedback:

1. I still prefer to have both `LB4` and `LB3` showing on the landing page, similar as the current one.
2. Can we have `Documentation` a first class tab instead of hiding inside `Resources`
3. Can we show github stars for both lb4 and lb3?

raymondfeng commented:
I see this error from the console:
```
[2019-05-30 13:39:05] ERROR `/lb3/images/overview/logo-collage.jpg' not found.
```

dhmlau commented:
> [2019-05-30 13:39:05] ERROR `/lb3/images/overview/logo-collage.jpg' not found.

I noticed that too. but couldn't find this jpg anywhere, I'll remove the reference of it to get rid of the error.

What about this now? 
<img width="1157" alt="Screen Shot 2019-05-30 at 7 45 25 PM" src="https://user-images.githubusercontent.com/25489897/58672169-96417400-8313-11e9-8f3c-20da7f4df3d5.png">

1. I've put back the `Docs` on the navbar
2. removed `LoopBack3` and github star button from the navbar
3. Added the LB3 and LB4 button. The buttons look a bit weird but I can fix the padding around it if you're good with this design.

dhmlau commented:
This is how it looks like now:
<img width="1222" alt="Screen Shot 2019-05-30 at 11 08 21 PM" src="https://user-images.githubusercontent.com/25489897/58679054-e4647080-832f-11e9-8cc1-662d3ee95e70.png">

raymondfeng commented:
I suggest that we get rid of LB4 GA banner. It’s not adding much value any more.

raymondfeng commented:
What about a shortcut for blogs? We are good at publishing blogs and they are very useful information?

raymondfeng commented:
@dhmlau The link for `Docs` is broken - http://127.0.0.1:4001/doc/en/lb.

nabdelgadir commented on css/loopback.css:
```diff
@@ -347,3 +362,17 @@ footer a:hover {
+}
+.btn-main:hover a.main-btn-link {
+  color: var(--color-blue-xd);
+}
```
Missing new line

dhmlau commented:
Sorry about that @raymondfeng @nabdelgadir. I did some more testing and found more issues. :(  
I think I've fixed them all.  Once this PR is approved, I will squash the commit. Thanks.

dhmlau commented:
@slnode test please
